Sources/Support/SDKTester.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SDKTester.swift
+//
+//  Created by Nacho Soto on 9/21/22.
+
+import Foundation
+
+/// `SDKTester` allows you to ensure that the SDK is set up correctly by diagnosing configuration errors.
+/// To run the test, simply call ``SDKTester/test()``.
+///
+/// #### Example:
+/// ```swift
+/// let tester = SDKTester.default
+/// do {
+///     try await tester.test()
+/// } catch {
+///     print("SDKTester failed: \(error.localizedDescription)")
+/// }
+/// ```
+@objc(RCSDKTester)
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
+public final class SDKTester: NSObject {
+
+    typealias SDK = PurchasesType & InternalPurchasesType
+
+    private let purchases: SDK
+
+    init(purchases: SDK) {
+        self.purchases = purchases
+    }
+
+    /// Default instance of `SDKTester`.
+    /// Note: you must call ``Purchases/configure(with:)-6oipy`` before using this.
+    @objc
+    public static let `default`: SDKTester = .init(purchases: Purchases.shared)
+}
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
+extension SDKTester {
+
+    /// An error that represents a failing step in ``SDKTester``
+    public enum Error: Swift.Error {
+
+        /// Connection to the API failed
+        case failedConectingToAPI(Swift.Error)
```

Comment:
There's a small typo here

```suggestion
        case failedConnectingToAPI(Swift.Error)
```

Reply:
Thanks.